dist/leaflet.distortableimage.js

```diff
@@ -207,8 +206,23 @@ L.EXIF = function getEXIFdata(img) {
         altitude = 0; // none
       }
     }
+
+      var panTo = new L.latLng(lat,lng);
+
+      var x_diff = overlay._corners[0].lng - overlay._corners[1].lng;  // width -- a
+      var y_diff = overlay._corners[0].lat - overlay._corners[2].lat;  // height -- b
+
+      overlay._corners[0] = new L.latLng(lat + y_diff/2, lng + x_diff/2);
+      overlay._corners[1] = new L.latLng(lat + y_diff/2, lng - x_diff/2);
+      overlay._corners[2] = new L.latLng(lat - y_diff/2, lng + x_diff/2);
+      overlay._corners[3] = new L.latLng(lat - y_diff/2, lng - x_diff/2);
+
+      overlay.editing._rotateBy(angle);
+
+      overlay._map.setView(panTo,14);
+
   } else {
-    alert("EXIF initialized. Press again to view data in console.");
+    alert("EXIF initialized. Press again to view data in console and locate to original EXIF coordinates.");
```

Comment:
Can we refactor to do this upon completion of getting the EXIF data out, rather than pressing a second time? This is going to be super!